src/forms/Household.js

```diff
@@ -223,10 +225,10 @@ const MemberField = function ({ household, time, setHousehold, updateClientValue
 
       <Columns.Two>
         <Role
-          member={ member }
-          setMember={ onMemberChange }
+          member           = { member }
+          setMember        = { onMemberChange }
           setDropdownMember={ onMemberDropdownChange }
```

Comment:
This and the surrounding changes don't quite match the style from above, but I'm not too bothered. If you're interested in changing it, you'd change this line to:
```js
setDropdownMember = { onMemberDropdownChange }
```
and then adjust the other lines to match.